mozilla/kuma
# Bug 1170235 compat tables

How to test:
- download this branch and compile your style sheets and compress assets
- create a macro called "Template:EmbedCompatTable" from this:  https://gist.github.com/darkwing/e6b94e895c7af6850682
- add the following function to "Template:mdn:common":  https://gist.github.com/darkwing/f1f004c098037bbe81fd
- create a document that uses the template like `{{ EmbedCompatTable('web-css-display') }}`

Still to do:
- [ ] Icon font not appearing (may be unrelated issue)
- [x] Space in the table slides open but the history drawer itself just appears.
- [x] Space in the table slides closed but the history drawer itself just dissipears.
- [x] The heading is also 'sliding open' when the history draw is opened in tablet/flex view. (I'll look at a fix)
- [x] Need explicit way to trigger thick border between support cells when medium switches.
- [x] Add `.bc-medium-border` to the macro
- [x] Add bug number to that one commit 
- [x] solve animation problem
- [x] decide if need waffle

Where should we identify problems with the macro? The contents, legend, and title of many of the icons/abbreviations need to be changed.

Supersedes #3258
